static/app/views/explore/logs/useLogsQuery.tsx

```diff
@@ -98,6 +99,12 @@ function useLogsQueryKey({limit, referrer}: {referrer: string; limit?: number})
   const {selection, isReady: pageFiltersReady} = usePageFilters();
   const location = useLocation();
   const projectIds = useLogsProjectIds();
+  const traceIds = baseSearch?.tokens
+    .filter(
+      token =>
+        token.key === OurLogKnownFieldKey.TRACE_ID && token.type === TokenType.FILTER
+    )
+    .map(token => token.value);
```

Comment:
We can maybe move traceids into a context for the embedded pages only, since this may conflict with a user searching for a trace on the explore page